### PR TITLE
Bq9 tela de autenticação

### PR DESCRIPTION
**Pull request by Jeansidharta**
[Trello card](https://trello.com/c/AkCiAuI4/7-bq-9-fazer-tela-de-autentica%C3%A7%C3%A3o)

Iphone5:
image

Reviewer: @RafaelDoering

### CODE REVIEW

File: src/pages/login/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React from 'react'
```
Comment (RafaelDoering) [inline src/pages/login/index.js:1]: cooltext345006849905477

File: src/pages/login/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      const password = passwordRef.current.value
+      event.preventDefault()
+      const action = await AuthActions.login({ email, password })
+
+      // This timeout is to prevent unmounting before the state changes
+      setTimeout(() => {
+        dispatch(action)
+        history.push(Routes.home)
+      })
+    } catch (e) { console.error(e) } finally {
```
Comment (RafaelDoering) [inline src/pages/login/index.js]: cooltext345007025306731

**Review by RafaelDoering — APPROVED** [review]

File: src/pages/login/index.js (hunk 1 above)
Comment (Jeansidharta) [reply src/pages/login/index.js:1]: Cool Text - ok 345007186550251

File: src/pages/login/index.js (hunk 2 above)
Comment (Jeansidharta) [reply src/pages/login/index.js]: Cool Text - ok 345007186550251

**Pull request merged**